spec/config/initializers/rack_attack_spec.rb

```diff
@@ -0,0 +1,122 @@
+# require 'rails_helper'
```

Comment:
I wouldn't push commented specs into the project. Is there a reason why this is not running or erased ?